packages/trpc/server/routers/loggedInViewer/connectedCalendars.schema.ts

```diff
@@ -3,6 +3,9 @@ import { z } from "zod";
 export const ZConnectedCalendarsInputSchema = z
   .object({
     onboarding: z.boolean().optional(),
+    // Fetches the calendars for this event-type only if present
+    // Otherwise, fetches the calendars for the authenticated user
+    eventTypeId: z.number().nullable(),
```

Comment:
```suggestion
    eventTypeId: z.number().nullish(),
```

Reply:
Discussed in DM as well.

I did this to ensure that the caller is fully aware of whether he is operating at user level or event-level.
Allowing undefined(which would be considered null automatically) might allow accidentally operating at user-elvel when infact it is just a mistake of not passing `eventTypeId`